joshuaCarpentier/quiz/queue1.js

```diff
@@ -0,0 +1,70 @@
+// Problem Statement
+// Given an array of positive numbers and a positive number ‘k,’ find the maximum sum of any contiguous subarray of size ‘k’.
+
+
+
+  /**
+   * 
+   * Input: Array of positive numbers & a positive num k
+   * Output: Sum
+   *    
+   * find sum of any contiguous subarray compare to k 
+   * 
+   * Naive Approach
+   * --------------- 
+   * Variables: 
+   * 
+   * 
+   */
+   console.log(`\n ------- Test Case -------- `);
+
+const maxSum = (arr, k) =>{
+    console.log(`${arr}`)
+    if(!arr.length) return k
+    
+
+
+    let totalSum =0;
```

Comment:
We also need some additional variables here to keep track of the start of our sliding window, the current window's sum and the maxSum we've found so far.